lib/v2/hackyournews/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'hackyournews.com': {
+        _name: 'HackYourNews',
+        '.': [
+            {
+                title: 'AI summaries of the top HN stories',
```

Comment:
```suggestion
                title: 'Index',
```
This follows the heading level 3 of the docs.